msc_pygeoapi/process/weather/__init__.py

```diff
@@ -40,6 +42,10 @@
     LOGGER.warning(err)
 
 
+GEOMET_DDI_BASEPATH = '/data/geomet/feeds/ddi'
```

Comment:
Since we may be able to reuse these for other / future providers agains the same storage base, abstract into `msc-pygeoapi.env`

```
MSC_PYGEOAPI_DATA_FEED_DDI
MSC_PYGEOAPI_DATA_FEED_SCIENCE
```